benchmark/cluster/README.md

```diff
@@ -0,0 +1,78 @@
+# Cluster Training Benchmark
+
+## Setup
+
+- Platform
+  - Kubernetes: v1.6.2
+  - Linux Kernel: v3.10.0
+
+- Resource
+  - CPU: 10 Cores per Pod
+  - Memory: 5GB per Pod
+
+- Docker Image
+
+  We use different base Docker Image to run the benchmark on Kubernetes:
+  - PaddlePaddle v2: paddlepaddle/paddle:[commit-id]
```

Comment:
v2 should use 0.10.0 tag, and fluid should use commit id

Reply:
Done, since 0.10.0 does not support distributed training, use 0.11.0 .